components/choose_download_action_dialog.py

```diff
@@ -0,0 +1,66 @@
+from PySide2.QtWidgets import QRadioButton
+from PySide.QtWidgets import QDialog, QDialogButtonBox, QVBoxLayout, QLabel
+import Utils
+from APIClient import ConnStatus
+
+logger = Utils.getLogger(__name__)
+
+
+class ChooseDownloadActionDialog(QDialog):
+    def __init__(self, name, data_parent, parent=None):
+        super().__init__(parent)
+        self.quit_on_close = False
+        conn_status = data_parent.api.getStatus()
+
+        self.setWindowTitle(name)
+        self.answer = 0  # 0 = cancel, 1 = open web, 2 = download to memory
+
+        message = QLabel(f"What should we do with '{name}'?")
+
+        buttons = QDialogButtonBox.Ok | QDialogButtonBox.Cancel
+        self.button_box = QDialogButtonBox(buttons)
+        self.button_box.accepted.connect(self.accept)
+        self.button_box.rejected.connect(self.reject)
+
+        if conn_status == ConnStatus.CONNECTED:
+            self.choose_open_on_web = QRadioButton("Explore on Ondsel Website")
```

Comment:
"Explore on the Ondsel Lens website", because I think that we should keep the distinction between the Ondsel website (ondsel.com) vs Lens (lens.ondsel.com)